Increase Screen Area on client and Agent Panel

EIJason opened the pull request:
These changes make use of most of the screen, so no wasted areas on the side also added a bootstrap to the client header to improve the look and feel slightly...

![image](https://user-images.githubusercontent.com/55503152/69737315-16625e00-112c-11ea-97ae-61ad4a38badf.png)

![image](https://user-images.githubusercontent.com/55503152/69737430-4c074700-112c-11ea-9d00-33722bf9be66.png)

EIJason committed:
Increase Screen Area on client and Agent Panel

These changes make use of most of the screen, so no wasted areas on the side also added a bootstrap to the client header to improve the look and feel slightly...

EIJason committed:
Allow \ In Auth Username Fix

If you logon using a Server\User then this fix would need to be applied so mail fetching can work...

JediKev commented on include/client/header.inc.php:
```diff
@@ -31,33 +31,36 @@
-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-plugins.js"></script>
-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-osticket.js"></script>
-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/select2.min.js"></script>
+    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js?f1e9e88"></script>
```
What's up with `?f1e9e88` after all of the URLs..?

EIJason committed:
Update header.inc.php

EIJason committed:
Update header.inc.php

EIJason committed:
Update header.inc.php

EIJason commented:
I am not sure where that came from I fixed it though?

From: JediKev <notifications@github.com>
Sent: 27 November 2019 15:44
To: osTicket/osTicket <osTicket@noreply.github.com>
Cc: Jason Kisogloo <jason@europeincoming.co.uk>; Author <author@noreply.github.com>
Subject: Re: [osTicket/osTicket] Increase Screen Area on client and Agent Panel (#5233)


@JediKev commented on this pull request.

________________________________

In include/client/header.inc.php<https://github.com/osTicket/osTicket/pull/5233#discussion_r351361359>:

>      <!-- Favicons -->

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-32x32.png" sizes="32x32" />

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-16x16.png" sizes="16x16" />

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-1.12.1.custom.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-timepicker-addon.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>js/osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/filedrop.field.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>scp/js/bootstrap-typeahead.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-plugins.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/select2.min.js"></script>

+    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js?f1e9e88"></script>

What's up with ?f1e9e88 after all of the URLs..?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/osTicket/osTicket/pull/5233?email_source=notifications&email_token=ANHOSMBBBQ6YYXAV64KZENTQV2IVTA5CNFSM4JSIU7OKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCNGFQKA#pullrequestreview-323770408>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANHOSMA4AXERVQUEGEGK5KLQV2IVTANCNFSM4JSIU7OA>.
I am not sure where that came from I fixed it though?

From: JediKev <notifications@github.com>
Sent: 27 November 2019 15:44
To: osTicket/osTicket <osTicket@noreply.github.com>
Cc: Jason Kisogloo <jason@europeincoming.co.uk>; Author <author@noreply.github.com>
Subject: Re: [osTicket/osTicket] Increase Screen Area on client and Agent Panel (#5233)


@JediKev commented on this pull request.

________________________________

In include/client/header.inc.php<https://github.com/osTicket/osTicket/pull/5233#discussion_r351361359>:

>      <!-- Favicons -->

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-32x32.png" sizes="32x32" />

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-16x16.png" sizes="16x16" />

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-1.12.1.custom.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-timepicker-addon.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>js/osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/filedrop.field.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>scp/js/bootstrap-typeahead.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-plugins.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/select2.min.js"></script>

+    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js?f1e9e88"></script>

What's up with ?f1e9e88 after all of the URLs..?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/osTicket/osTicket/pull/5233?email_source=notifications&email_token=ANHOSMBBBQ6YYXAV64KZENTQV2IVTA5CNFSM4JSIU7OKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCNGFQKA#pullrequestreview-323770408>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANHOSMA4AXERVQUEGEGK5KLQV2IVTANCNFSM4JSIU7OA>. 

Jason Kisogloo
IT Systems Specialist
@|from|department|@
Europe Incoming Ltd & Tradetrav Ltd - @|from|st|@

Direct No: +44 (0) 203 994 0446
Office No: +44 (0) 208 994 5001

Fax No: +44 (0) 208 994 5044
Skype ID: jasonkis
Email: jason@europeincoming.co.uk

Address: Unit 11-12 Turnham Green Terrace Mews, London, W4 1QU, England
      
Follow or like us on
Facebook https://www.facebook.com/europeincoming.co.uk?ref=hl
LinkedIn https://www.linkedin.com/authwall?trk=ripf&trkInfo=AQF8ahG9cSWaGwAAAWbkg8Gg5EtxVxyZ7qKU7HCVbfzF08n4m9ilv9tvaVH1ysXeH1-wVY0emz8BjcCcT7sX4VZ8X84vDvr8Nlt1s-VICM5vRp3uwBKswaP1PYitR-Apa7eWV5Q=&originalReferer=&sessionRedirect=https%3A%2F%2Fwww.linkedin.com%2Fin%2Feurope-incoming-136985156%2F
Instagram https://www.instagram.com/europeincoming/
Twitter https://twitter.com/europe_incoming

Europe Incoming Ltd is registered in England & Wales. Company number: 7053949.
Registered office: Charterhouse (Accountants) Ltd, 166 College Road, Middlesex, HA1 1RA
For our Terms & Conditions please visit www.europeincoming.com or www.fitbookings.com
For our Privacy Policy please visit www.europeincoming.com/privacy-policy/ 

Offices: England | France | Italy | Scotland | Switzerland | Scandinavia

www.europeincoming.com | www.fitbookings.com | www.micebookings.com


This E-mail is from Europe Incoming Ltd. The E-mail and any files transmitted with it are confidential and intended solely for the use of the individuals to which they are addressed. Any unauthorized dissemination or copying of this E-mail, and any use or disclosure of any attachments contained in it, is strictly prohibited and may be illegal. If you have received the E-mail in error please notify sales@europeincoming.co.uk or telephone +44 208 994 5001. Please do not copy or disclose its contents to any person or body, and delete it from your computer systems. Europe Incoming Ltd recommends that this email and all attachments are checked for viruses before use. No responsibility is accepted by Europe Incoming Ltd for any loss or damage arising in any way from receipt or use thereof. Europe Incoming Ltd is registered in England and Wales with no. 7053949. A list of directors is available at the registered office. Registered Office: Charterhouse (Accountants) Ltd, 166 College Road, Middlesex, HA1 1RA

EIJason reviewed (commented):
Ok this is my proposed changes both for the interface expansion and the mailfetch if username is in the format Server\Username

EIJason commented:
Hi JediKev,

I have a small request, can you add a Job title field into the agents that can be put on the response templates as the agent name can be?

I am using a “ – “ in the Surname and adding it following that, but the field isn’t long enough for some….

Thanks
Jason

From: JediKev <notifications@github.com>
Sent: 27 November 2019 15:44
To: osTicket/osTicket <osTicket@noreply.github.com>
Cc: Jason Kisogloo <jason@europeincoming.co.uk>; Author <author@noreply.github.com>
Subject: Re: [osTicket/osTicket] Increase Screen Area on client and Agent Panel (#5233)


@JediKev commented on this pull request.

________________________________

In include/client/header.inc.php<https://github.com/osTicket/osTicket/pull/5233#discussion_r351361359>:

>      <!-- Favicons -->

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-32x32.png" sizes="32x32" />

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-16x16.png" sizes="16x16" />

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-1.12.1.custom.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-timepicker-addon.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>js/osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/filedrop.field.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>scp/js/bootstrap-typeahead.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-plugins.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/select2.min.js"></script>

+    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js?f1e9e88"></script>

What's up with ?f1e9e88 after all of the URLs..?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/osTicket/osTicket/pull/5233?email_source=notifications&email_token=ANHOSMBBBQ6YYXAV64KZENTQV2IVTA5CNFSM4JSIU7OKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCNGFQKA#pullrequestreview-323770408>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANHOSMA4AXERVQUEGEGK5KLQV2IVTANCNFSM4JSIU7OA>.
Hi JediKev,

I have a small request, can you add a Job title field into the agents that can be put on the response templates as the agent name can be?

I am using a “ – “ in the Surname and adding it following that, but the field isn’t long enough for some….

Thanks
Jason

From: JediKev <notifications@github.com>
Sent: 27 November 2019 15:44
To: osTicket/osTicket <osTicket@noreply.github.com>
Cc: Jason Kisogloo <jason@europeincoming.co.uk>; Author <author@noreply.github.com>
Subject: Re: [osTicket/osTicket] Increase Screen Area on client and Agent Panel (#5233)


@JediKev commented on this pull request.

________________________________

In include/client/header.inc.php<https://github.com/osTicket/osTicket/pull/5233#discussion_r351361359>:

>      <!-- Favicons -->

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-32x32.png" sizes="32x32" />

     <link rel="icon" type="image/png" href="<?php echo ROOT_PATH ?>images/oscar-favicon-16x16.png" sizes="16x16" />

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-1.12.1.custom.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-ui-timepicker-addon.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>js/osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/filedrop.field.js"></script>

-    <script src="<?php echo ROOT_PATH; ?>scp/js/bootstrap-typeahead.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor.min.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-plugins.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/redactor-osticket.js"></script>

-    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/select2.min.js"></script>

+    <script type="text/javascript" src="<?php echo ROOT_PATH; ?>js/jquery-3.4.0.min.js?f1e9e88"></script>

What's up with ?f1e9e88 after all of the URLs..?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/osTicket/osTicket/pull/5233?email_source=notifications&email_token=ANHOSMBBBQ6YYXAV64KZENTQV2IVTA5CNFSM4JSIU7OKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCNGFQKA#pullrequestreview-323770408>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANHOSMA4AXERVQUEGEGK5KLQV2IVTANCNFSM4JSIU7OA>. 

Jason Kisogloo
IT Systems Specialist
@|from|department|@
Europe Incoming Ltd & Tradetrav Ltd - @|from|st|@

Direct No: +44 (0) 203 994 0446
Office No: +44 (0) 208 994 5001

Fax No: +44 (0) 208 994 5044
Skype ID: jasonkis
Email: jason@europeincoming.co.uk

Address: Unit 11-12 Turnham Green Terrace Mews, London, W4 1QU, England
      
Follow or like us on
Facebook https://www.facebook.com/europeincoming.co.uk?ref=hl
LinkedIn https://www.linkedin.com/authwall?trk=ripf&trkInfo=AQF8ahG9cSWaGwAAAWbkg8Gg5EtxVxyZ7qKU7HCVbfzF08n4m9ilv9tvaVH1ysXeH1-wVY0emz8BjcCcT7sX4VZ8X84vDvr8Nlt1s-VICM5vRp3uwBKswaP1PYitR-Apa7eWV5Q=&originalReferer=&sessionRedirect=https%3A%2F%2Fwww.linkedin.com%2Fin%2Feurope-incoming-136985156%2F
Instagram https://www.instagram.com/europeincoming/
Twitter https://twitter.com/europe_incoming

Europe Incoming Ltd is registered in England & Wales. Company number: 7053949.
Registered office: Charterhouse (Accountants) Ltd, 166 College Road, Middlesex, HA1 1RA
For our Terms & Conditions please visit www.europeincoming.com or www.fitbookings.com
For our Privacy Policy please visit www.europeincoming.com/privacy-policy/ 

Offices: England | France | Italy | Scotland | Switzerland | Scandinavia

www.europeincoming.com | www.fitbookings.com | www.micebookings.com


This E-mail is from Europe Incoming Ltd. The E-mail and any files transmitted with it are confidential and intended solely for the use of the individuals to which they are addressed. Any unauthorized dissemination or copying of this E-mail, and any use or disclosure of any attachments contained in it, is strictly prohibited and may be illegal. If you have received the E-mail in error please notify sales@europeincoming.co.uk or telephone +44 208 994 5001. Please do not copy or disclose its contents to any person or body, and delete it from your computer systems. Europe Incoming Ltd recommends that this email and all attachments are checked for viruses before use. No responsibility is accepted by Europe Incoming Ltd for any loss or damage arising in any way from receipt or use thereof. Europe Incoming Ltd is registered in England and Wales with no. 7053949. A list of directors is available at the registered office. Registered Office: Charterhouse (Accountants) Ltd, 166 College Road, Middlesex, HA1 1RA

JediKev commented:
@EIJason

You mean like adding a Custom Form Field and then using the form field variable in the response template? That’s already a thing via Admin Panel > Manage > Forms and Admin Panel > Emails > Templates.

Cheers.

EIJason commented:
Nope I mean When adding Agents I Add a job title to the Agents Surname following a Hyphen...

As we cannot add custom fields to the Agent Creation Dialogue correct? I checked manage forms…

What is preferable is a job title in the Create / Edit Agent Form

Then the ability to add that Agents name (as you can now) and title to a template (agents title not available yet)

Does that make sense?

Regards
Jason

From: JediKev <notifications@github.com>
Sent: 28 November 2019 15:55
To: osTicket/osTicket <osTicket@noreply.github.com>
Cc: Jason Kisogloo <jason@europeincoming.co.uk>; Mention <mention@noreply.github.com>
Subject: Re: [osTicket/osTicket] Increase Screen Area on client and Agent Panel (#5233)


@EIJason<https://github.com/EIJason>

You mean like adding a Custom Form Field and then using the form files variable in the response template? That’s already a thing via Admin Panel > Manage > Forms and Admin Panel > Emails > Templates.

Cheers.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/osTicket/osTicket/pull/5233?email_source=notifications&email_token=ANHOSMGIFSSB2EBEWSQAYYDQV7SUPA5CNFSM4JSIU7OKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFM7FSQ#issuecomment-559542986>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANHOSMGXRE4SDVSXQY3T223QV7SUPANCNFSM4JSIU7OA>.
Nope I mean When adding Agents I Add a job title to the Agents Surname following a Hyphen...

As we cannot add custom fields to the Agent Creation Dialogue correct? I checked manage forms…

What is preferable is a job title in the Create / Edit Agent Form

Then the ability to add that Agents name (as you can now) and title to a template (agents title not available yet)

Does that make sense?

Regards
Jason 

Jason Kisogloo
IT Systems Specialist
@|from|department|@
Europe Incoming Ltd & Tradetrav Ltd - @|from|st|@

Direct No: +44 (0) 203 994 0446
Office No: +44 (0) 208 994 5001

Fax No: +44 (0) 208 994 5044
Skype ID: jasonkis
Email: jason@europeincoming.co.uk

Address: Unit 11-12 Turnham Green Terrace Mews, London, W4 1QU, England
      
Follow or like us on
Facebook https://www.facebook.com/europeincoming.co.uk?ref=hl
LinkedIn https://www.linkedin.com/authwall?trk=ripf&trkInfo=AQF8ahG9cSWaGwAAAWbkg8Gg5EtxVxyZ7qKU7HCVbfzF08n4m9ilv9tvaVH1ysXeH1-wVY0emz8BjcCcT7sX4VZ8X84vDvr8Nlt1s-VICM5vRp3uwBKswaP1PYitR-Apa7eWV5Q=&originalReferer=&sessionRedirect=https%3A%2F%2Fwww.linkedin.com%2Fin%2Feurope-incoming-136985156%2F
Instagram https://www.instagram.com/europeincoming/
Twitter https://twitter.com/europe_incoming

Europe Incoming Ltd is registered in England & Wales. Company number: 7053949.
Registered office: Charterhouse (Accountants) Ltd, 166 College Road, Middlesex, HA1 1RA
For our Terms & Conditions please visit www.europeincoming.com or www.fitbookings.com
For our Privacy Policy please visit www.europeincoming.com/privacy-policy/ 

Offices: England | France | Italy | Scotland | Switzerland | Scandinavia

www.europeincoming.com | www.fitbookings.com | www.micebookings.com


This E-mail is from Europe Incoming Ltd. The E-mail and any files transmitted with it are confidential and intended solely for the use of the individuals to which they are addressed. Any unauthorized dissemination or copying of this E-mail, and any use or disclosure of any attachments contained in it, is strictly prohibited and may be illegal. If you have received the E-mail in error please notify sales@europeincoming.co.uk or telephone +44 208 994 5001. Please do not copy or disclose its contents to any person or body, and delete it from your computer systems. Europe Incoming Ltd recommends that this email and all attachments are checked for viruses before use. No responsibility is accepted by Europe Incoming Ltd for any loss or damage arising in any way from receipt or use thereof. Europe Incoming Ltd is registered in England and Wales with no. 7053949. A list of directors is available at the registered office. Registered Office: Charterhouse (Accountants) Ltd, 166 College Road, Middlesex, HA1 1RA
From: JediKev <notifications@github.com>
Sent: 28 November 2019 15:55
To: osTicket/osTicket <osTicket@noreply.github.com>
Cc: Jason Kisogloo <jason@europeincoming.co.uk>; Mention <mention@noreply.github.com>
Subject: Re: [osTicket/osTicket] Increase Screen Area on client and Agent Panel (#5233)


@EIJason<https://github.com/EIJason>

You mean like adding a Custom Form Field and then using the form files variable in the response template? That’s already a thing via Admin Panel > Manage > Forms and Admin Panel > Emails > Templates.

Cheers.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/osTicket/osTicket/pull/5233?email_source=notifications&email_token=ANHOSMGIFSSB2EBEWSQAYYDQV7SUPA5CNFSM4JSIU7OKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEFM7FSQ#issuecomment-559542986>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANHOSMGXRE4SDVSXQY3T223QV7SUPANCNFSM4JSIU7OA>.

EIJason commented:
![image](https://user-images.githubusercontent.com/55503152/69820071-319a9f80-11f8-11ea-9311-1e857711715b.png)

![image](https://user-images.githubusercontent.com/55503152/69820102-437c4280-11f8-11ea-929f-6cec9080bb8e.png)

EIJason committed:
Update theme.css
